src/patternfly/layouts/Flex/flex.scss

```diff
@@ -116,276 +13,285 @@ $pf-flex-spacer-map: (
 
-    &.pf-m-justify-content-flex-end#{$mod} {
-      justify-content: flex-end;
-    }
+      &.pf-m-justify-content-flex-end#{$breakpoint-name} {
+        justify-content: flex-end;
+      }
 
-    &.pf-m-justify-content-center#{$mod} {
-      justify-content: center;
-    }
+      &.pf-m-justify-content-center#{$breakpoint-name} {
+        justify-content: center;
+      }
 
-    &.pf-m-justify-content-space-between#{$mod} {
-      justify-content: space-between;
-    }
+      &.pf-m-justify-content-space-between#{$breakpoint-name} {
+        justify-content: space-between;
+      }
 
-    &.pf-m-justify-content-space-around#{$mod} {
-      justify-content: space-around;
-    }
+      &.pf-m-justify-content-space-around#{$breakpoint-name} {
+        justify-content: space-around;
+      }
 
-    &.pf-m-justify-content-space-evenly#{$mod} {
-      justify-content: space-evenly;
-    }
+      &.pf-m-justify-content-space-evenly#{$breakpoint-name} {
+        justify-content: space-evenly;
+      }
 
-    // align-items
-    &.pf-m-align-items-flex-start#{$mod} {
-      align-items: flex-start;
-    }
+      // align-items
+      &.pf-m-align-items-flex-start#{$breakpoint-name} {
+        align-items: flex-start;
+      }
 
-    &.pf-m-align-items-flex-end#{$mod} {
-      align-items: flex-end;
-    }
+      &.pf-m-align-items-flex-end#{$breakpoint-name} {
+        align-items: flex-end;
+      }
 
-    &.pf-m-align-items-center#{$mod} {
-      align-items: center;
-    }
+      &.pf-m-align-items-center#{$breakpoint-name} {
+        align-items: center;
+      }
 
-    &.pf-m-align-items-stretch#{$mod} {
-      align-items: stretch;
-    }
+      &.pf-m-align-items-stretch#{$breakpoint-name} {
+        align-items: stretch;
+      }
 
-    &.pf-m-align-items-baseline#{$mod} {
-      align-items: baseline;
-    }
+      &.pf-m-align-items-baseline#{$breakpoint-name} {
+        align-items: baseline;
+      }
 
-    // align-content
-    &.pf-m-align-content-flex-start#{$mod} {
-      align-content: flex-start;
-    }
+      // align-content
+      &.pf-m-align-content-flex-start#{$breakpoint-name} {
+        align-content: flex-start;
+      }
 
-    &.pf-m-align-content-flex-end#{$mod} {
-      align-content: flex-end;
-    }
+      &.pf-m-align-content-flex-end#{$breakpoint-name} {
+        align-content: flex-end;
+      }
 
-    &.pf-m-align-content-center#{$mod} {
-      align-content: center;
-    }
+      &.pf-m-align-content-center#{$breakpoint-name} {
+        align-content: center;
+      }
 
-    &.pf-m-align-content-stretch#{$mod} {
-      align-content: stretch;
-    }
+      &.pf-m-align-content-stretch#{$breakpoint-name} {
+        align-content: stretch;
+      }
 
-    &.pf-m-align-content-space-between#{$mod} {
-      align-content: space-between;
+      &.pf-m-align-content-space-between#{$breakpoint-name} {
+        align-content: space-between;
 
-      > * {
-        margin-right: 0;
-        margin-left: 0;
+        > * {
```

Comment:
Can we remove this? It creates this

<img width="518" alt="Screen Shot 2019-09-10 at 4 14 00 PM" src="https://user-images.githubusercontent.com/35148959/64650990-1634ab00-d3e6-11e9-8a29-3ae1312cbb8d.png">

instead of this

<img width="518" alt="Screen Shot 2019-09-10 at 4 13 52 PM" src="https://user-images.githubusercontent.com/35148959/64651001-1a60c880-d3e6-11e9-889c-7883d17ad28a.png">

Looks fine in a column layout.